helix-term/src/commands.rs

```diff
@@ -390,6 +391,38 @@ impl PartialEq for Command {
 
 fn no_op(_cx: &mut Context) {}
 
+fn raise_key(cx: &mut Context, key_event: KeyEvent) {
+    let callback = Box::pin(async move {
+        let call: job::Callback =
+            Box::new(move |editor: &mut Editor, compositor: &mut Compositor| {
```

Comment:
I wonder if it's really worth doing this: it'll introduce extra latency because you're essentially returning, calling callback after the editor loop to spawn a new asynchronous task, then triggering the compositor again (note that the task might also be arbitrarily delayed by LSP traffic).

Reply:
Good point, let's close this for now until we need have the need.